codecov.yml

```
@@ -6,3 +6,7 @@ coverage:
     patch:
       default:
         informational: true
+ignore:
+  # Examples implementatio and test code should not count against coverage.
```

Why shouldn't `tests/grpc` count against coverage? Everything in a testing dir should be part of a test, and thus count against coverage.

```suggestion
  # Examples implementation and test code should not count against coverage.
```

---

As I understand it, it's a part of tests, not an object being tested. Do we want to see the percentage of lines covered in that folder?

---

I think it's useful as a sanity check on the tests. If there are tests that have low coverage percentages, its a good indicator that something has gone wrong in testing.

I don't feel strongly either way, but I do think that code coverage on the tests themselves is helpful.

---

I don't feel strong either - I'll return it to err on the side of more data